src/filters.rs

```diff
@@ -1253,63 +1253,59 @@ impl Filter for WorkspaceView {
 #[grammar = "view_parser.pest"]
 struct MyParser;
 
-fn make_view(cmd: &str, name: &str) -> Box<dyn Filter> {
-    if cmd == "+" || cmd == "prefix" {
-        return Box::new(PrefixView {
-            prefix: Path::new(name).to_owned(),
-        });
-    } else if cmd == "hide" {
-        return Box::new(HideView {
-            path: Path::new(name).to_owned(),
-        });
-    } else if cmd == "empty" {
-        return Box::new(EmptyView);
-    } else if cmd == "info" {
-        let mut s = std::collections::BTreeMap::new();
-        for p in name.split(",") {
-            let x: Vec<String> = p.split("=").map(|x| x.to_owned()).collect();
-            if x.len() == 2 {
-                s.insert(x[0].to_owned(), x[1].to_owned());
-            } else {
-                s.insert("prefix".to_owned(), x[0].to_owned());
-            }
+fn kvargs(args: &[&str]) -> std::collections::BTreeMap<String, String> {
```

Comment:
feels a bit overkill for arguments but why not

Reply:
Yes it is :)